src/main/java/pages/CustomerLoginPage.java

```diff
@@ -0,0 +1,29 @@
+package pages;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+
+public class CustomerLoginPage {
+
+    private static WebDriver driver;
+
+    public CustomerLoginPage(WebDriver driver) {
+        this.driver = driver;
+    }
+
+    public void customerLoginAs (String email, String password) {
+        WebElement emailInput = driver.findElement(By.name("login[username]"));
```

Comment:
We're trying to use POO. In that case element should be extracted as class member and then reuse in the method.
So above private static WebDriver driver; all elements should be defined using approach private final By element = By.tag(element).

Example for this class:
`private final By emailInput = By.name("login[username]");
private final By passwordInput = By.name("login[password]");
private final By signInButton = By.id("send2");`

The reason behind this is to be able to reuse defined elements in many methods in the class.